modular_newhorizons/master_files/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -0,0 +1,24 @@
+/singleton/reagent/toxin/potassium_chlorophoride
+	description = "A specific chemical based on Potassium Chloride to stop the heart for surgery. Not safe to eat!"
+
+/singleton/reagent/lexorin
+	description = "Lexorin temporarily stops respiration. Causes tissue damage."
+
+/singleton/reagent/soporific
+	description = "An effective hypnotic used to treat insomnia, can act as a sedative. Lasts three times longer when inhaled."
+
+/singleton/reagent/polysomnine
+	name = "Chloral Hydrate"
+	description = "A powerful sedative. Lasts two times longer when inhaled."
+
+/singleton/reagent/slimetoxin
+	description = "A corruptive toxin produced by slimes."
```

Comment:
Proposal, to change to the current Aurora version, effectively removing the override basically, though the above is also fine.


" A transformative toxin isolated from jelly extract from green slimes. Use of the chemical has profound effects on the body’s cells, converting animal cells into unique slime cells. These slime cells begin to replace the normal cells of the body, resulting in the development of ‘slime people’, though eventually these degenerate into grey slimes"